unit_test/blas/Test_Blas1_iamax.hpp

```diff
@@ -13,9 +13,7 @@ namespace Test {
     typedef typename AT::mag_type mag_type;
 
     typedef Kokkos::View<ScalarA*[2],
-       typename std::conditional<
-                std::is_same<typename ViewTypeA::array_layout,Kokkos::LayoutStride>::value,
-                Kokkos::LayoutRight, Kokkos::LayoutLeft>::type,Device> BaseTypeA;
+       typename ViewTypeA::array_layout,Device> BaseTypeA;
```

Comment:
The purpose of this line before was to make a BaseTypeA with 2 columns, so that its first column could either be LayoutStride or contiguous. But in practice it's never LayoutStride because it's guarded by ``KOKKOSKERNELS_INST_LAYOUTSTRIDE``, which isn't a macro that we ever define in KokkosKernels_config.h. And the tests aren't allowed to call non-ETI'd versions of kernels. So I'm not sure what to do about these tests.

Reply:
@kyungjoo-kim

Reply:
I'm in favor of allowing unit tests to call non-ETI combinations sometimes, rather than inst'ing the whole library for LayoutStride.